frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -120,9 +122,22 @@ const ClutchApp: React.FC<ClutchAppProps> = ({
     [workflowSessionStore]
   );
 
-  const appContextValue = React.useMemo(() => ({ workflows: discoverableWorkflows }), [
-    discoverableWorkflows,
-  ]);
+  const appContextValue = React.useMemo(
+    () => ({
+      workflows: discoverableWorkflows,
+      triggerHeaderItem: (item: HeaderItems, open: boolean, data: unknown) =>
+        // Will set the open status and spread any additional data onto the property named after the item
+        setTriggeredHeaderData({
+          ...triggeredHeaderData,
```

Comment:
should we rename these to include `feedback` or `nps` or see this as being reusable for non-feedback flows?

Reply:
I made this initially so it could be reused for non-feedback flows, if we think it wont ever be reused though I can rename and strongly type it though